web/html/src/branding/css/susemanager/components/buttons.scss

```diff
@@ -11,6 +11,14 @@
   }
 }
 
+.btn {
+  &,
+  &:active,
+  &:first-child:active {
+    border-color: transparent;
```

Comment:
Is this valid for both Uyuni and the Suma theme? I haven't deployed this change and tested it, mostly curious since this looks like it could be a problem on Uyuni.

Reply:
Yes, it works fine for both themes.